spec/features/admin_can_view_order_details_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "rails_helper"
+require "factory_helper"
+
+feature "admin user" do
+  scenario "can view individual orders" do
+    category = Category.create(
+      name: "Plants",
+      description: "Carnivorous plants!"
+    )
+
+    plant1 = category.products.create(name: "Plant 1",
+                        description: "This is the description for plant 1",
+                        price:       19.99,
+                        image_url:   "plants/plant-2.jpg",
+                        status:      "active")
+    plant2 = category.products.create(name: "Plant 2",
+                        description: "This is the description for plant 2",
+                        price:       29.99,
+                        image_url:   "plants/plant-2.jpg",
+                        status:      "active")
+
+    user = User.create(first_name: "Jane",
+                       last_name:  "Doe",
+                       email:      "jane@doe.com",
+                       password:   "password")
+
+    order = user.orders.create(status: "ordered",
+                       created_at: DateTime.civil(2015, 07, 05, 21, 33, 0),
+                       updated_at: DateTime.civil(2015, 07, 05, 21, 33, 0))
+
+    order.order_items.create(product_id: plant1.id,
+                             quantity:   3,
+                             unit_price: plant1.price)
+    order.order_items.create(product_id: plant2.id,
+                             quantity:   1,
+                             unit_price: plant2.price)
+
+
+    admin = User.create(first_name: "Mike",
+                        last_name:  "D",
+                        email:      "mike@mike.com",
+                        password:   "12345678",
```

Comment:
Looks good. Appears lines 28-29 would spill over the 80 character limit if you moved it right.